requirements.txt

```diff
@@ -0,0 +1 @@
+mbed-host-tests>=1.5.0,<2
```

Comment:
Can we explain in the "Add setup.py for greentea repo" commit message what we're doing with setup.py, and why we're merging htrun and greentea into the same package?